python_modules/libraries/dagster-embedded-elt/dagster_embedded_elt_tests/dlt_tests/conftest.py

```diff
@@ -0,0 +1,28 @@
+import os
+from typing import Generator, Tuple
+
+import dlt
+import pytest
+from dlt import Pipeline
+from dlt.extract.source import DltSource
+
+from .constants import EXAMPLE_PIPELINE_DUCKDB
+from .dlt_test_sources.duckdb_with_transformer import pipeline
+
+
+@pytest.fixture(autouse=True, scope="module")
+def setup_dlt_pipeline() -> Generator[Tuple[DltSource, Pipeline], None, None]:
+    if os.path.exists(EXAMPLE_PIPELINE_DUCKDB):
+        os.remove(EXAMPLE_PIPELINE_DUCKDB)
```

Comment:
Consider removing the constant, parameterizing with `testrun_uuid`, and making the fixture function scoped.

That way, we don't need the constant and we can enable `pytest-xdist` on this test suite.

Reply:
Ended up using a `uuid` in the fixture for the dlt `pipeline_name` -- this is what determines the `.duckdb` file. Curious what you think.